docs/src/build_tips.md

```diff
@@ -115,7 +115,6 @@ The following environment variables are automatically set in the build environme
 * `CXX`: the C++ cross compiler
 * `FC`: the Fortran cross compiler
 * `OBJC`: the Objective-C cross compiler
-* `AR`: the archiver
```

Comment:
Wait, if we don't set `$AR` anymore that's probably a bug; we should, I think.  :P

Reply:
Well, I thought [this was done on purpose](https://github.com/JuliaPackaging/Yggdrasil/commit/2a9ef87c95b704cb78684c0fa58cea21b3a15db8) :stuck_out_tongue:

Reply:
Okay; I think in general it doesn't matter _that_ much, because we now have the compiler wrappers that live at the default names.  But if we're going to set all those other environment variables, we should set `AR` as well for consistency's sake, I think.

The real power is in `HOSTCC` and whatnot pointing to `x86_64-linux-musl-gcc`.

Reply:
Also `LD` is gone, is that expected? :thinking:

Reply:
Okay if they're all gone then I guess I decided they weren't needed anymore, so we should remove them all.

Reply:
OK, `{{done}}`!